docs/docs/cloud/reference/api/openapi.json

```diff
@@ -2457,6 +2446,17 @@
         },
         "tags": ["MCP"]
       },
+      "get": {
```

Comment:
This might be a tricky thing to maintain since this file is copied directly from the source in `openapi.json` in `langgraph-api`. Can you update it there as well?